### PR TITLE
PAS-536 | Add support for `AuthenticatorDisplayName`

### PR DESCRIPTION
**Pull request by Tyrrrz**
Waiting on: https://github.com/bitwarden/passwordless-dotnet/pull/152

### CODE REVIEW

**Review by abergs — COMMENTED** [review]
We need to add migrations, corect?

**Comment by Tyrrrz** [comment]
@abergs yes, correct

**Comment by Tyrrrz** [comment]
This needs a new release of Passwordless.NET and a package update to resolve missing references

**Comment by Tyrrrz** [comment]
Added migrations and updated the Passwordless.NET SDK version to `2.1.0-beta.3` which is pending approval here: https://github.com/bitwarden/passwordless-devops/actions/runs/10775849985

Once that package is released, this PR should technically be good to merge. Would still need to run some smoke tests for good measure, however.

File: src/Service/Fido2Service.cs [hunk 1]
```diff
@@ -16,53 +16,32 @@
 
 namespace Passwordless.Service;
 
-public class Fido2Service : IFido2Service
+public class Fido2Service(
```
Comment (Tyrrrz) [inline src/Service/Fido2Service.cs:19]: Only formatting changes here

**Comment by Tyrrrz** [comment]
Should be ready

**Review by jonashendrickx — APPROVED** [review]

**Pull request merged**